plugin-api/src/main/java/org/sonar/api/measures/CoreMetrics.java

```diff
@@ -964,6 +964,39 @@ public final class CoreMetrics {
     .setOptimizedBestValue(true)
     .create();
 
+  /**
+   * @since 10.4
+   */
+  public static final String NEW_ACCEPTED_ISSUES_KEY = "new_accepted_issues";
+
+  /**
+   * @since 10.4
+   */
+  public static final Metric<Integer> NEW_ACCEPTED_ISSUES = new Metric.Builder(NEW_ACCEPTED_ISSUES_KEY, "New Accepted Issues", Metric.ValueType.INT)
+    .setDescription("New accepted issues")
+    .setDirection(Metric.DIRECTION_WORST)
+    .setDomain(DOMAIN_ISSUES)
+    .setBestValue(0.0)
+    .setOptimizedBestValue(true)
+    .setDeleteHistoricalData(true)
```

Comment:
why do we have a difference between the two measures on `setDeleteHistoricalData`?

Reply:
I'm following what is done for other metrics with a "NEW" version. I think it makes sense to keep historical data for base metrics, not so much for "NEW" ones because they are related to short-living versions or PR, which is why it's done like this.

Reply:
ok make sense